core/commands/destroy_command.rb

```diff
@@ -102,29 +102,33 @@ def filter_nodes_by_name(vm_list, node_name)
   def display_all_nodes
     vagrant_cleaner = VagrantCleaner.new(@env, @ui)
     vagrant_vm_list = vagrant_cleaner.vm_list
-    aws_vm_list = @aws_service.instances_names_list
+    aws_vm_list = @aws_service&.instances_names_list || []
```

Comment:
We should discuss: should it be easier to provide `aws_service` all the time?

Reply:
In order not to check for the existence of the service, but in order to return empty data if the service is not configured?